src/components/MoviesList/MoviesList.js

```diff
@@ -1,6 +1,27 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList = () => (
-  <>Put the list here</>
+export const MoviesList = ({ moviesProperties }) => (
+  <div className="movies">
+    {moviesProperties.map(card => (
```

Comment:
```suggestion
    {moviesProperties.map(movie => (
```

It is not a card it is a movie